### DISCUSSION TITLE
Issues related to installing milvus in k8s

### DISCUSSION THREAD

**Discussion by cschan123** [milvus-io/milvus#31975] [Q&A and General discussion]
When installing milvus in k8s, its components etcd and mimio prompt running, while the body miluvs displays a Crashloopback status prompt. The error message is as follows: CrashLoopBackOff (back-off 5m0s restarting failed container=standalone pod=milvus-standalone-bd4d44cb8-lbtnc_rqa-server(75dfb049-db78 -4140-ac54-9a4233654dc9)) | Last state: Terminated with 2: Error, started: Mon, Apr 8 2024 9:24:32 am, finished: Mon, Apr 8 2024 9:24:32 am; while continuing Restart a lot of times
![63de43802f1bc61d1458c8ec4862784](https://github.com/milvus-io/milvus/assets/57937229/63ddcd89-3d1f-4444-a586-359bef0c6b73)
![3ad1adbfecc736112f6cb24609af6d7](https://github.com/milvus-io/milvus/assets/57937229/dcc97211-a115-4cd4-98c7-13d30dc45eca)

**Comment by yhmo** [comment]
Are you using Milvus v2.0.0? It is a pretty old version of 3 years ago.
The latest stable version is v2.3.13
